### PR TITLE
Clean an reorganize the repo

### PR DESCRIPTION
**Pull request by sam-pires**
- Organized the repo
- Added some guidelines in the readme.md and contributing.md files

### CODE REVIEW

**Comment by sam-pires** [comment]
Ping @LouiseBonnard for review :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,6 +70,26 @@ That's it: you have contributed to this open source project! Congratulations!
...
+## Rules to follow when creating specs
+### File Naming
+Use dashes `-` for separators in images names and use lowercase.To help navigating through specifications please follow the structure:
+`{{parent category}}-{{parent category}}-{{page name}}`
+Example:
+`catalog-products-add-edit.md`
+`orders-orders-list.md`
+
+
+This allows to easily where the page is located, especially as some pages have the same name but are located in different places, with a totally different purpose.
```
Comment (LouiseBonnard) [inline CONTRIBUTING.md:82]:
```suggestion
This allows to easily spot where the page is located, especially as some pages have the same name but are located in different places, with a totally different purpose.
```

File: README.md [hunk 2]
```diff
@@ -95,11 +95,17 @@ When not possible, template can be adapted, but it is important to have the info
...
 ## Repository tree
 
-The repository has 3 main directories, Back-office, Front-office and images.
+The repository has 4 main directories, `back-office`, `front-office`, `broader-topics` and `modules`.
 
-The `Back-office` (BO) directory mostly follows the back-office structure. Most files contains the specifications for a BO page.
+- The `back-office` directory mostly follows the back-office structure. Most files contains the specifications for a BO page.
 
-The `Front-office` directory list the pages in the Front-office and shared behaviors throughout the pages.
+- The `front-office` directory list the pages in the front-office and shared behaviors throughout the pages.
```
Comment (LouiseBonnard) [inline README.md:102]:
```suggestion
- The `front-office` directory lists the pages in the front office and shared behaviors throughout the pages.
```

File: README.md [hunk 3]
```diff
@@ -95,11 +95,17 @@ When not possible, template can be adapted, but it is important to have the info
 
 ## Repository tree
 
-The repository has 3 main directories, Back-office, Front-office and images.
+The repository has 4 main directories, `back-office`, `front-office`, `broader-topics` and `modules`.
 
-The `Back-office` (BO) directory mostly follows the back-office structure. Most files contains the specifications for a BO page.
+- The `back-office` directory mostly follows the back-office structure. Most files contains the specifications for a BO page.
```
Comment (LouiseBonnard) [inline README.md:100]:
```suggestion
- The `back-office` directory mostly follows the back office structure. Most files contain the specifications for a BO page.
```

**Comment by sam-pires** [comment]
@LouiseBonnard you can merge when you ready

**Pull request merged**